site/app/templates/polls/PollForm.twig

```diff
@@ -225,33 +226,34 @@
         toggleTimerInputs();
     };
     function toggleTimerInputs() {
-        var enableTimerCheckbox = document.getElementById("enable-timer");
-        var timerInputs = document.getElementById("timer-inputs").querySelectorAll(".poll-timer");
+        const enableTimerCheckbox = $("#enable-timer");
+        const timerInputs = $(".poll-timer");
 
-        if (enableTimerCheckbox.checked) {
-            document.getElementById("timer-inputs").style.display = "block";
-            timerInputs.forEach(function(input) {
-                input.setAttribute("type", "number");
-                input.setAttribute("min", "0");
-                if (input.classList.contains("poll-hours")) {
-                    input.setAttribute("placeholder", "Hour");
-                    input.setAttribute("max", "24");
-                } else if (input.classList.contains("poll-minutes")) {
-                    input.setAttribute("placeholder", "Min");
-                } else if (input.classList.contains("poll-seconds")) {
-                    input.setAttribute("placeholder", "Sec");
+        if (enableTimerCheckbox.prop("checked")) {
+            $("#timer-inputs").show();
+            timerInputs.each(function() {
+                $(this).attr("type", "number")
+                    .attr("min", "0");
+                if ($(this).hasClass("poll-hours")) {
+                    $(this).attr("placeholder", "Hour")
+                        .attr("max", "24");
+                } else if ($(this).hasClass("poll-minutes")) {
+                    $(this).attr("placeholder", "Min");
+                } else if ($(this).hasClass("poll-seconds")) {
+                    $(this).attr("placeholder", "Sec");
                 }
```

Comment:
Why do you need to do this?  Can't you just add these in the HTML markup directly?

Reply:
Fixed